src/win/conpty.cc

```diff
@@ -193,11 +201,9 @@ static NAN_METHOD(PtyStartProcess) {
     shellpath = filename;
   }
 
-  std::string shellpath_(shellpath.begin(), shellpath.end());
-
   if (shellpath.empty() || !path_util::file_exists(shellpath)) {
     std::stringstream why;
```

Comment:
Using a stringstream makes more sense to me because we pass a non-wide string to it, and then later call `.str().c_str()` on it. If we used a wstringstream, we'd have to convert the non-wide string to a wide string, and then somehow convert the resulting string back when calling [ThrowError](https://github.com/nodejs/nan/blob/dd5edf87e84abb678befe754cd55c7b99326248a/doc/errors.md#nanthrowerror), which doesn't take wide chars.

Reply:
You would have to call `from_wstring(shellpath)` in either cases, instead it can be

```
std::stringstream why;
why << "File not found: " << shellpath;
Nan::ThrowError(from_wstring(why.str()).c_str());
```